lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -296,6 +293,17 @@ sub scenario_name {
     return $scenario;
 }
 
+# searches for the testsuite where the name matches our TEST setting ignoring "+..." suffix
```

Comment:
But the "+…" suffix is just a convention, these are still individual testsuites. I am not sure this is what we want

Reply:
If that's not what we want, then I remove the first commit considering the current behavior is fine after all. (The 2nd one which adds an additional check is likely still useful.)